website/templates/util/render_node.mako

```diff
@@ -78,7 +78,7 @@
                     <dt><span class="date log-date" data-bind="text: log.date.local, tooltip: {title: log.date.utc}"></span></dt>
                     <dd class="log-content">
                         <span data-bind="if:log.anonymous">
-                            <span><em>A user</em></span>
+                            <span data-bind="html: anonymousUserName"></span>
```

Comment:
Instead of binding here, try just adding the `scripted` class to an element that is a descendant of the element to which the Knockout binding is applied.

Elements with the `scripted` class are hidden by default. The binding is ultimately applied with a call to `$osf.applyBindings`, which finds children of the element with the `scripted` class and makes them visible.

https://github.com/CenterForOpenScience/osf.io/blob/5da70fc7dabb1bee298580c569f02bf2828c5e90/website/static/js/osfHelpers.js#L463-L465